ariadne/cardano/src/Ariadne/Wallet/Knit.hs

```diff
@@ -327,6 +337,9 @@ selectCommandName = "select"
 sendCommandName :: CommandId
 sendCommandName = "send"
 
+calcTotalName :: CommandId
+calcTotalName = "sum-coins"
```

Comment:
The convention of this module is that all `CommandId`s have name `***CommandName`. I propose to be consistent with that.

Reply:
I meant that there is the word `Command` in all these names, so I would call it `sumCoinsCommandName`. Just to be consistent with naming in this module.

Reply:
Oops, my bad